src/config/scrivitoSites.ts

```diff
@@ -37,8 +37,55 @@ export function siteForUrl(
 
 export async function ensureSiteIsPresent() {
   if ((await load(currentSiteId)) === null) {
-    navigateTo(() =>
-      Obj.onAllSites().where('_path', 'equals', '/').order('_language').first(),
-    )
+    navigateTo(() => {
+      const sites = Obj.onAllSites().where('_path', 'equals', '/').toArray()
```

Comment:
Does this also include neoletter content - if available?

Reply:
Yes, currently all multi-site related code assumes all sites are on the same origin and their content is suited for the Portal App. In order to support non-portal sites we'd two things need to be done:

* Update the multi-site config to properly handle "non-portal" sites, e.g., Neoletter content or Example App content
* In the `ensureSiteIsPresent` helper only consider base URLs that match the pattern `<current_origin>/<site_language>`

Reply:
Could you takle this in a follow up PR?

Reply:
Part two, sure!
Any idea if and how we could implement part 1?:
> to properly handle "non-portal" sites

Reply:
For me it would be "good enough" to filter out known other site names such as Neoletter. Not a generic "there can be endless many sites" (yet).